examples/complete/main.tf

```diff
@@ -135,10 +139,10 @@ module "eks_cluster" {
 
 module "eks_node_group" {
   source  = "cloudposse/eks-node-group/aws"
-  version = "2.4.0"
+  version = "2.10.0"
 
   subnet_ids                    = module.this.enabled ? module.subnets.public_subnet_ids : ["filler_string_for_enabled_is_false"]
-  cluster_name                  = module.eks_cluster.eks_cluster_id
+  cluster_name                  = coalesce(module.eks_cluster.eks_cluster_id, "disabled")
```

Comment:
Wouldn't it be better to use a `count` expression that doesn't instance nodes until the cluster is set up?